package.json

```diff
@@ -45,17 +46,18 @@
     "web3-utils": "1.2.6"
   },
   "devDependencies": {
-    "@openzeppelin/test-helpers": "0.5.10",
     "@openeth/truffle-typings": "0.0.6",
-    "@types/node": "^18.0.0",
-    "@typescript-eslint/eslint-plugin": "~4.28.3",
-    "@typescript-eslint/parser": "~4.28.3",
+    "@openzeppelin/test-helpers": "0.5.10",
```

Comment:
remove if not used

Reply:
They are used inside of other classes

Reply:
<img width="331" alt="Screenshot 2022-08-26 at 2 27 03 pm" src="https://user-images.githubusercontent.com/36888576/186914194-0c565a84-232b-4095-b57f-893b00915456.png">
it sure doesn't look like it

Reply:
updated